apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx

```diff
@@ -33,27 +34,46 @@ const ComponentStatus = () => {
     components,
   } = data.allComponentStatusYaml.edges[0].node
 
+  const { Head, Body, Row, Cell } = Table
+
   return (
     <Table summary={summary}>
-      <thead>
-        <Tr>
-          {headers.map((text, index) => (
-            <ThH key={camelify(text)} scope="col">
+      <Head>
+        <Row>
+          {headers.map((text) => (
+            <Cell
+              as="th"
+              style={{ textAlign: 'left' }}
+              key={camelify(text)}
+              scope="col"
+            >
               {text}
-            </ThH>
+            </Cell>
           ))}
-        </Tr>
-      </thead>
-      <tbody>
+        </Row>
+      </Head>
+      <Body>
         {components.map((component, index) => (
-          <Tr key={component.component + index}>
-            <ThV key={component.component + index} scope="row">
-              <Link to={`/components/${kebabify(component.component)}`}>
+          <Row key={component.component + index}>
+            <Cell
+              as="th"
+              key={component.component + index}
+              scope="row"
+              style={{
+                textAlign: 'left',
```

Comment:
here to

Reply:
Actually, it seems this was solved by setting `text-align: left` on `thead` – so the vertical th’s are still centered.

Reply:
Fixed in #454